.github/workflows/e2e.yml

```
@@ -2,10 +2,28 @@ name: End-to-end tests
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - '**.md'
+      - '**.d.ts'
+      - 'examples/**'
+      - 'private/**'
+      - 'website/**'
   pull_request_target:
     types: [ opened, synchronize, reopened, labeled ]
+    paths-ignore:
+      - '**.md'
+      - '**.d.ts'
+      - 'examples/**'
+      - 'private/**'
+      - 'website/**'
```

Let's add this one since it's already covered by `pull_request` event.
```suggestion
      - 'website/**'
      - .github/workflows/e2e.yml
```